content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -160,9 +160,7 @@ flags `--etcd-certfile=k8sclient.cert`, `--etcd-keyfile=k8sclient.key` and
 `--etcd-cafile=ca.cert`.
 
 {{< note >}}
-etcd authentication is not currently supported by Kubernetes. For more
-information, see the related issue
-[Support Basic Auth for Etcd v2](https://github.com/kubernetes/kubernetes/issues/23398).
+etcd authentication is not planned for Kubernetes.
```

Comment:
Personally I think it's fine to remove the link, but will be good to see what others think, if this is good background context to keep in place.

Reply:
Maybe we should keep it. It atleast tells people "Oh, it was request but there's been a _decision_ not to support it."

Reply:
We [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future)

How about a follow up PR to rephrase?